### PR TITLE
Journal week 4

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by jeremyrist** [comment]
:+1: Great journal! I agree 110% :)

**Comment by karjac14** [comment]
Life is good! Our assignments are getting harder as it goes but we can all do this! :+1:

File: journal-week-4.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Last week, last week. A lot of ups and downs last week. 
+
+I was very happy understanding CSS and getting results from the page. Colors, font styles and boxing contents, 
+that was fun until it gets complicated with links and grids that open and closed and lists that you need to reorganize all elements. 
+
+I have some merge conflicts last week too, at the end and after following some directions, I solved them with no huge problems. 
+It gives you an enormous level of satisfaction when finally you can solve your own problems,. Get your hands dirty and once they are very messy, you can figure it out.
+
```
Comment (kellymurray) [inline journal-week-4.md:8]: :+1:

File: journal-week-4.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Now we are putting almost all the pieces together, HTML, CSS and JS together and it looks very cool!!
+
+At the same time, we have sad news. Emily is not coming back, she left the program. That was the saddest part of the week.
```
Comment (kellymurray) [inline journal-week-4.md:11]: I agree :cry:

**Comment by jmcreasman** [comment]
Good entries :+1: - I was sad too to see Emily go

**Pull request merged**